spec/latest/core/index.html

```diff
@@ -627,10 +627,64 @@ <h3>Documenting a Web API</h3>
 
     <p>Aforementioned <i>title</i> and <i>description</i> SHOULD take precedence
       over standard <i>rdfs:label</i> and <i>rdfs:comment</i>.</p>
+
+    <p>There is one more relish related to how Linked Data works. Consider the
```

Comment:
Relish?

![Relish](https://user-images.githubusercontent.com/12283/72760808-4ed96200-3bda-11ea-979f-8cb3e35d97f2.jpg)

Reply:
Why not? :). Anyway, I've reworded it